src/Backend/Modules/Faq/Actions/Add.php

```diff
@@ -80,13 +80,12 @@ private function validateForm(): void
             if ($this->form->isCorrect()) {
                 // build item
                 $item = [];
-                $item['meta_id'] = $this->meta->save();
+                $item['meta'] = $this->meta;
```

Comment:
Do not change anything in the actions, only in the model.

Reply:
Hmm, how do you handle the meta then?

Reply:
In the model.php

Reply:
But you'll only have an meta_id, no entity?

Reply:
Fixed 232348709